test/cbmc/proofs/DNS/DNSgetHostByName_a/DNSgetHostByName_a_harness.c

```diff
@@ -47,8 +118,8 @@
 * function.
 ****************************************************************/
 
-uint32_t prvParseDNSReply( uint8_t * pucUDPPayloadBuffer,
-                           size_t xBufferLength,
+uint32_t __CPROVER_file_local_FreeRTOS_DNS_c_prvParseDNSReply( uint8_t * pucUDPPayloadBuffer,
+                           size_t xBufferLength,struct freertos_addrinfo **ppxAddressInfo,
```

Comment:
Doesn't your style guide require a new line and indention here?

Reply:
It does... I was going to merge a commit :)